docs/source/using-ood/interactiveapps.rst

```diff
@@ -44,35 +44,116 @@ On the Interactive Apps tab, you can view the list of available interactive apps
   If you already have a Notebook saved to your home directory, on the **Files** tab,
   click the name of the file to restart the Notebook in a new tab on your browser.
 
-Working with Jupyter Notebook [Custom Anaconda Environment]
-============================================================
-One of the interactive apps on OOD is Jupyter Notebook Custom Anaconda Environment. You need to do some initial
+Working with JupyterLab Notebook
+================================
+
+One of the interactive apps on OOD is JupyterLab Notebook. You need to do some initial
 setup in order to use this app effectively. This section will provide a walkthrough of setting up and using this app.
 The general workflow is to first create a virtual python environment; ensure that Jupyter Notebook is installed in your virtual
-environment; and then reference this environment when you start the Jupyter Notebook [Custom Anaconda Environment] OOD interactive app.
+environment; and then reference this environment when you start the JupyterLab Notebook OOD interactive app.
+
+You can find the JupyterLab notebook on OOD by following these steps:
+
+1. Go to https://ood.discovery.neu.edu/
+2. Click on Interactive Apps
+3. Select JupyterLab Notebook from the dropdown list
+
+
+The OOD form for launching JupyterLab notebook looks like this:
+
+.. image:: /images/jupyterlab_form.PNG
+
+In the remainder of this section we will go over each of these fields.
+
+Working Directory
+------------------
+
+This field contains the absolute file path to the directory from where the JupyterLab instance will be started. 
+In the image above, the working directory is set to /home/k.joisher (By default the working directory is set to the home directory of the user).
+On launching the JupyterLab instance with this working directory, we can see the contents of the home directory in the file explorer 
+
+.. image:: /images/jupyterlab_workingdirectory.PNG
+
+Partition
+----------
+
+This field contains the partition on which the JupyterLab session would run. Different partitions have different limits on the time, CPU and memory resources.
+You can find more information about these partitions on the following link: https://rc-docs.northeastern.edu/en/latest/hardware/partitions.html .
+
+If you want to use GPU for your JupyterLab session then select the gpu; or multigpu if you have access to that partition. Once you select either of these partitions, additional fields will be displayed on the form which ask for 
+GPU type, CUDA version and number of gpus. You can find additional information about the various GPU types on the following page: https://rc-docs.northeastern.edu/en/latest/using-discovery/workingwithgpu.html .
```

Comment:
Adding clickable hyperlink would be more readable: `following page <https://rc-docs.northeastern.edu/en/latest/using-discovery/workingwithgpu.html>`_